test/webdriverio/specs/jira.js

```diff
@@ -0,0 +1,173 @@
+describe("Jira", function () {
+
+  var bugTrackerUrl = 'https://jira.atlassian.com';
```

Comment:
remove constants that are used only once. Move constants to test script for better readability